lib/screen/view_playlist/view_playlist.dart

```diff
@@ -0,0 +1,406 @@
+                      ),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                            playList?.name ?? tr('untitled'),
+                            style: playList?.name == null
+                                ? theme.textTheme.atlasSpanishGreyBold36
+                                : theme.textTheme.headline1,
+                          ),
+                          Padding(
+                            padding: const EdgeInsets.only(bottom: 40, top: 5),
+                            child: Text(
+                              tr(
+                                  tokensPlaylist.length > 1
+                                      ? 'artworks'
+                                      : 'artwork',
+                                  args: [tokensPlaylist.length.toString()]),
+                              style: theme.textTheme.atlasBlackMedium12,
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                    Expanded(
+                      child: BlocConsumer<NftCollectionBloc,
+                          NftCollectionBlocState>(
+                        bloc: nftBloc,
+                        builder: (context, nftState) {
+                          return NftCollectionGrid(
+                            state: nftState.state,
+                            tokens: nftState.tokens,
+                            loadingIndicatorBuilder: loadingView,
+                            customGalleryViewBuilder: (context, tokens) =>
+                                assetsWidget(
+                              context,
+                              setupPlayList(
+                                tokens: tokens,
+                                selectedTokens: playList?.tokenIDs,
+                              ),
+                              accountIdentities: accountIdentities,
+                            ),
+                          );
+                        },
+                        listener: (context, nftState) {},
+                      ),
+                    )
+                  ],
+                ),
+              ),
+              Positioned(
+                bottom: 0,
+                left: 0,
+                right: 0,
+                child: PlaylistControl(
+                  onPlayTap: () {
+                    final payload = ArtworkDetailPayload(
+                      accountIdentities,
+                      0,
+                      isPlaylist: true,
+                    );
+                    Navigator.of(context).pushNamed(
+                      AppRouter.artworkPreviewPage,
+                      arguments: payload,
+                    );
+                  },
+                ),
+              ),
+            ],
+          ),
+        );
+      },
+    );
+  }
+}
+
+Widget assetsWidget(
```

Comment:
This function look like the same in `add_new_playlist.dart`. Can we create a widget then reuse?